debian/control

```diff
@@ -4,7 +4,7 @@ Priority: extra
 Maintainer: Andrey Sibiryov <kobolog@yandex-team.ru>
 Build-Depends: cmake, cdbs, debhelper (>= 7.0.13), libcocaine-dev (>= 0.11.0.0),
  libboost-filesystem-dev, libboost-thread-dev, 
- libswarm-dev (>= 0.6.1.0), libswarm-dev (<< 0.7.0),
+ libswarm-dev (>= 0.6.1.0), libswarm-dev (<< 0.7.4),
```

Comment:
I think `libswarm-dev (>= 0.7.0)` should be enough, at least until there are no new swarm versions appeared.